DigitalLearningSolutions.Web/ViewModels/LearningMenu/PostLearningAssessmentViewModel.cs

```diff
@@ -0,0 +1,41 @@
+﻿namespace DigitalLearningSolutions.Web.ViewModels.LearningMenu
+{
+    using DigitalLearningSolutions.Data.Models.PostLearningAssessment;
+
+    public class PostLearningAssessmentViewModel
+    {
+
+        public string CourseTitle { get; }
+        public string SectionName { get; }
+        public string PostLearningAssessmentPath { get; }
+        public bool PostLearningLocked { get; }
+        public string AssessmentStatus { get; }
+        public int CustomisationId { get; }
+        public int SectionId { get; }
+
+        public PostLearningAssessmentViewModel(PostLearningAssessment postLearningAssessment, int customisationId, int sectionId)
+        {
+            CourseTitle = postLearningAssessment.CourseTitle;
+            SectionName = postLearningAssessment.SectionName;
+            PostLearningAssessmentPath = postLearningAssessment.PostLearningAssessmentPath;
+            PostLearningLocked = postLearningAssessment.PostLearningLocked;
+
+            if (postLearningAssessment.PostLearningAttempts == 0)
```

Comment:
I'd prefer for this to be extracted to avoid reassigning to the same variable and making the constructor overly complicated 🙂